webpack/base.js

```diff
@@ -32,6 +32,8 @@ module.exports = {
     new ExtractTextPlugin('style.css', {
       allChunks: true
     }),
+    new webpack.NoErrorsPlugin(),
+    new webpack.optimize.OccurenceOrderPlugin(),
```

Comment:
I just learned that if we specify `-p` on the webpack command line, it will do this (and minimize the output) for us without including the plugin.  Is that a better option @lexun?